tools/travis/setupPrereq.sh

```diff
@@ -32,5 +32,13 @@ $ANSIBLE_CMD couchdb.yml
 $ANSIBLE_CMD initdb.yml
 $ANSIBLE_CMD wipe.yml
 
+# deploy es for unittest
+$ANSIBLE_CMD elasticsearch.yml
+export ELASTIC_PROTOCOL="http"
+export ELASTIC_HOSTS="localhost:9200"
+export ELASTIC_INDEX_PATTERN="unittest-%s"
+export ELASTIC_USERNAME="admin"
+export ELASTIC_PASSWORD="admin"
+
```

Comment:
I'm not sure whether it is ok to add codes here, but without these lines, tests for `ElasticSearchActivationStore` will be skipped and it can introduce some risk to the system

Reply:
For test we can also look into using [TestContainers ElasticSearch][1] support. Then you would not need to deploy an ES container via ansible just for test runs

[1]: https://www.testcontainers.org/modules/elasticsearch/

Reply:
thanks, this looks great! I will look into it

Reply:
while using TestContainers, the gradle process will be time out cause of `GC overhead limit exceed` issue

Reply:
@jiangpengcheng could you please give more details? Even better if you can push a branch.
Testcontainers is not very memory greedy, so I believe this is something else

Reply:
@bsideup sorry to misleading you, actually the error is related to my environment, after do `./gradlew clean`, it worked